dbt_subprojects/daily_spellbook/models/_projects/fantasy/fantasy_events.sql

```diff
@@ -2,37 +2,15 @@
         schema = 'fantasy',
         alias = 'events',
         materialized = 'view',
-        tags=['static'],
-        post_hook='{{ hide_spells() }}'
+        post_hook = '{{ expose_spells(
+                        blockchains = \'["blast", "base"]\',
+                        spell_type = "project",
+                        spell_name = "events",
```

Comment:
```suggestion
                        spell_name = "fantasy",
```

Reply:
alias aka table name is always displayed, this will help differentiate that it's fantasy events